problem-spring-web/README.md

```diff
@@ -19,14 +19,19 @@ Add the starter module to your dependencies. That is all you will need to get a
 The autoconfiguration will configure problem-spring-web to handle all problems plus Spring Security problems if Spring Security is detected
 
 ### WebMVC
-If you're not using Spring Boot, add the following dependency to your project:
+If you're not using Spring Boot, add the following dependencies to your project:
```

Comment:
I'm using Spring boot, but still need to add the starter. Guess it should be stated "If you are not using problem-spring-web-autoconfigure"

Reply:
Ah, I guess I mixed those up, did overlook the `problem-spring-web-starter` while commenting. So it should be "If you are not using problem-spring-web-starter"

Reply:
But would there be a good reason if you are using Spring Boot to not use the starter?

Reply:
There was no active decision against the starter in our case. We started our project with [Jhipster](https://www.jhipster.tech/) back then and they are not using it. I wasn't aware of the starter until the recent update, to be honest ... Guess I would use it if I'd start a new project from scratch as it eliminates lots of duplicate code.